go.mod

```diff
@@ -3,15 +3,16 @@ module github.com/parca-dev/parca-load
 go 1.18
 
 require (
-	github.com/bufbuild/connect-go v1.3.0
+	buf.build/gen/go/parca-dev/parca/connectrpc/go v1.11.0-20230823175648-8e6b94c22fce.1
```

Comment:
The new BSR Go module proxy splits plugin output into separate modules (think packages).

There are pros and cons to this, but this is a net new behaviour to look out for.